lib/Sabberworm/CSS/Rule/Rule.php

```diff
@@ -152,11 +154,17 @@ public function getIsImportant() {
 		return $this->bIsImportant;
 	}
 
+	/**
+	 * {@inheritdoc}
+	 */
 	public function __toString() {
-		return $this->render(new \Sabberworm\CSS\OutputFormat());
+		return $this->render(new OutputFormat());
 	}
 
-	public function render(\Sabberworm\CSS\OutputFormat $oOutputFormat) {
+	/**
+	 * {@inheritdoc}
+	 */
+	public function render(OutputFormat $oOutputFormat) {
```

Comment:
A Rule should also output its comments. Maybe we should be able to configure in `OutputFormat` which comments we want to output: only document-level, document-and-ruleset-level, document,-ruleset-and-rule-level.